source/includes/inkling-reference/_concept.html.md

```diff
@@ -17,33 +17,33 @@ A concept statement describes what the computer will learn. It can be a feature
 
 ### How Do I Use It?
 
-A typical concept statement is shown to the right:
-
 ```inkling
 concept conceptName
-  is (classifier | estimator)
+  is classifier                 # or 'is estimator'
   predicts (outputSchema)
   follows preceedingConcept1, input(schemaName)
   feeds output, subsequentconcept
 end
 ```
 
+A typical concept statement is shown to the right. (Select the Inkling tab.) Its structure and keywords
```

Comment:
If someone reads our docs on mobile the code snippets/syntax show below the text.

How about "A typical concept statement is shown in the Inkling tab."